core/src/components/alert/test/basic/index.html

```diff
@@ -228,7 +228,8 @@
           inputs: [
             {
               type: 'radio',
-              label: 'Radio 1',
+              label:
```

Comment:
Could we do this in a separate test or move the disabled radio so that it doesn't fall off the screenshot? Maybe since `Radio 6` already has long text that wraps, we could move that and the disabled one up?

```javascript
{
  type: 'radio',
  label: 'Radio 1',
  value: 'value1',
},
{
  type: 'radio',
  label: 'Radio 2',
  value: 'value2',
  disabled: true,
},
{
  type: 'radio',
  label: 'Radio 3 Radio 3 Radio 3 Radio 3 Radio 3 Radio 3 Radio 3 Radio 3 Radio 3 Radio 3',
  value: 'value3',
  checked: true,
},
{
  type: 'radio',
  label: 'Radio 4',
  value: 'value4',
},
{
  type: 'radio',
  label: 'Radio 5',
  value: 'value5',
},
{
  type: 'radio',
  label: 'Radio 6',
  value: 'value6',
},
```

Reply:
Fixed in https://github.com/ionic-team/ionic-framework/pull/27898/commits/fb3d0b621d0c429650ed64a80612a6f971e1eff1. Screenshots are updating now.